pkg/clusteragent/autoscaling/workload/telemetry.go

```diff
@@ -17,11 +22,39 @@ const (
 
 var commonOpts = telemetry.Options{NoDoubleUnderscoreSep: true}
 
-// rolloutTriggered tracks the number of patch requests sent by the patcher to the kubernetes api server
-var rolloutTriggered = telemetry.NewCounterWithOpts(
-	subsystem,
-	"rollout_triggered",
-	[]string{"owner_kind", "owner_name", "namespace", "status"},
-	"Tracks the number of patch requests sent by the patcher to the kubernetes api server",
-	commonOpts,
+func resourceListToString(m map[corev1.ResourceName]resource.Quantity) string {
+	var sb strings.Builder
+	for k, v := range m {
+		sb.WriteString(fmt.Sprintf("%s:%s", string(k), v.String()))
+	}
+	return sb.String()
+}
+
+var (
+	// rolloutTriggered tracks the number of patch requests sent by the patcher to the kubernetes api server
+	rolloutTriggered = telemetry.NewCounterWithOpts(
+		subsystem,
+		"rollout_triggered",
+		[]string{"owner_kind", "owner_name", "namespace", "status"},
+		"Tracks the number of patch requests sent by the patcher to the kubernetes api server",
+		commonOpts,
+	)
+
+	// telemetryHorizonalScale tracks the horizontal scaling recommendation values
+	telemetryHorizonalScale = telemetry.NewCounterWithOpts(
```

Comment:
Note that you need to differentiate metrics that can only be emitted by the leader and add the `le.JoinLeaderLabel` (you can check how it works in the externalmetrics package).